pandas/tests/io/json/test_pandas.py

```diff
@@ -1671,3 +1671,27 @@ def test_to_s3(self, s3_resource):
         assert target_file in (
             obj.key for obj in s3_resource.Bucket("pandas-test").objects.all()
         )
+
+    def test_json_pandas_na(self):
+        # GH 31615
+        result = pd.DataFrame([[pd.NA]]).to_json()
+        assert result == '{"0":{"0":null}}'
+
+    def test_json_pandas_na_label(self):
+        # GH 31615
+        result = pd.DataFrame({pd.NA: ["a"]}).to_json()
+        assert result == '{"<NA>":{"0":"a"}}'
+
+    def test_json_pandas_nulls(self, nulls_fixture):
```

Comment:
@simonjayhawkins do you know of an easy way to add `pd.NA` to `nulls_fixture` inline during injection? Or would adding another fixture on top be the only way?

Reply:
since we already have two nulls_fixtures (nulls_fixture and unique_nulls_fixture), I think adding another could be the simplest way for now. 

I think, in general, simpler to skip than to add a extra parameter (although I've not kept up with the latest pytest releases).

There is not a huge number of tests using the nulls_fixture and maybe tests that do should also test for pandas.NA, either now of in the future. So an alternative could be to add pd.NA to the existing nulls fixture and either skip of xfail.